scripts/spack/packages/spheral/package.py

```diff
@@ -203,3 +197,14 @@ def cmake_args(self):
         spec = self.spec
 
         return options
+
+    @property
+    def build_dirname(self):
+        """Directory name to use when building the package."""
+        return "spack-build-%s" % self.pkg.spec.dag_hash(7)
+
+    @property
+    def build_directory(self):
+        """Full-path to the directory to use when building the package."""
+        return os.path.join(self.pkg.stage.path, self.build_dirname)
```

Comment:
Is this to get `dev_path` to work what is this?

Reply:
This is in preparation to moving to spack environments. Forgot I left it in there. I can get rid of it but it shouldn't make a difference.

Reply:
We can leave it in if you can tell me what `dag_hash(7)` means

Reply:
Sounds good